swap/src/env.rs

```diff
@@ -75,13 +75,13 @@ impl GetConfig for Testnet {
     fn get_config() -> Config {
         Config {
             bitcoin_lock_confirmed_timeout: 12.hours(),
-            bitcoin_finality_confirmations: 1,
-            bitcoin_avg_block_time: 5.minutes(),
+            bitcoin_finality_confirmations: 3,
```

Comment:
Given we have a max swap limit of USD ~1k I'd be happy to reduce the number of min confirmations to 1 or 2. This would speed up the whole swap experience.

Reply:
We could change both `testnet` and `mainnet` to `2` finality confirmations by default. That would effect both CLI and ASB. 
I am not sure how big of a security assumption this actually is. Have you done the math? 
Note that for the ASB we allow configuring the `finality_confirmations` in the config as well, but don't expose it in the wizzard, see https://github.com/comit-network/xmr-btc-swap/pull/490/commits/13bbdd5d6e59c7998b4907a5a2bb5467fb197051

Reply:
>  I am not sure how big of a security assumption this actually is. Have you done the math?

Not really, I would fee safe with 2 confirmations up to a $10k limit. 

It is really hard to calculate but the hardware cost alone for this would be above $25billion. 
Meaning, I doubt we will have a 51% attack. Nevertheless, there is the risk of orphaned blocks which do happen every ~1000blocks. 2 orphaned blocks are extremely rare though. 

note: the calculation in https://www.crypto51.app/ are wrong

Reply:
If we go for Bitcoin  = 2 blocks by default then I would reduce Monero to `10` blocks as well (so we have an alignment on the work input for `20` mins for both). 
Monero enforces `10`by `10` unlock blocks being part of consensus. That's one reason why I would also not go lower with Bitcoin, otherwise it feels unbalanced.

I'll set Bitcoin to `2` and Monero to `10` finality confirmation blocks :)